### DISCUSSION TITLE
Individual Contributors?

### DISCUSSION THREAD

**Discussion by tmcclell** [Green-Software-Foundation/community_wg#54] [General]
I would like for the Community WG to vote on whether we can allow individual contributors to participate in GSF either through working groups, volunteers, and/or meetups.

**Comment by adamj89** [comment]
Sharing my initial thoughts on this - this would be a positive move, with care taken to ensure the lines between individuals and their companies is not too blurred. 

A positive outcome of this could be ICs influencing their organisation to become GSF members. It would also benefit self-employed folks.
